src/figuresClasses.ts

```diff
@@ -1,11 +1,83 @@
-export interface Figure {}
+import { error } from 'console';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public shape: 'triangle',
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (!['red', 'blue', 'green'].includes(color)) {
+      throw new Error('wrong color');
```

Comment:
[CODE STYLE] - The error message should be more descriptive. Consider specifying what colors are valid.

Reply:
Fix it according to AI comment